core/icons/icon.ts

```diff
@@ -15,6 +15,12 @@ import {Size} from '../utils/size.js';
 import {Svg} from '../utils/svg.js';
 import type {IconType} from './icon_types.js';
 
+/**
+ * The abstract icon class. Icons are visual elements that live in the top-start
+ * corner of the block. Usually they provide more "meta" information about a
+ * block (such as warnings or comments) as opposed to fields, which provide
+ * "actual" information, related to how a block functions.
```

Comment:
Observation: I have emacs configured to wrap ("auto-fill") comments at column 70, which allows a little slack in case future edits slightly lengthen some of the lines (i.e., without forcing a reflow).  (I'd not break a 70–80 character comment unnecessarily, though.)